src/components/Alerts/index.js

```diff
@@ -1,32 +1,67 @@
 import React from 'react';
 import marked from 'marked';
 import classNames from 'classnames';
+import hashSum from 'hash-sum';
 
 import alerts from '../../../alerts.json';
 import styles from './index.module.scss';
+import XSvg from '../../images/ex.svg';
 
-const Alert = ({message, important}) => (
-    <div
-        className={classNames({
-            [styles.alert]: true,
-            [styles.important]: important,
-        })}
-    >
-        <div
-            className={styles.container}
-            dangerouslySetInnerHTML={{
-                __html: marked(message),
-            }}
-        />
-    </div>
-);
+class Alert extends React.Component {
+    state = {dismissed: false};
+    componentDidMount() {
```

Comment:
Here we mount the component and later we set his state.

Maybe we could try to set the initial state in the constructor: https://reactjs.org/docs/react-component.html#constructor so that we will not see the alerts flickering when reloading the page.

Reply:
Normally I would, but that will not work for a server-rendered application. Remember, Gatsby generates a static HTML page during the build, and the browser API is obviously not available then. The JavaScript loads after the fact (it's why Gatsby sites are so fast). We can either include the alerts in the HTML and remove them dynamically on the client, or not include them add them in dynamically. There are no other options.

In any case, any flickering will only occur on the first page load; after that, pages are loaded via JavaScript.

Reply:
Oops, just realized I forgot to change the initial state to be dismissed. My point above still stands, but now the alerts will appear a moment after the page loads if the user hasn't dismissed them, rather than flashing visible for a second if the user has dismissed them (the alerts are not included in the static HTML any more).